### PR TITLE
remove helper function mult_frac

### PR DESCRIPTION
**Pull request by mcgibbon**
## Purpose

Helper function `mult_frac` was previously needed because we couldn't write `a = <expression involving a>`. This has since been fixed in gt4py, so we can remove the helper function.

Also renamed cmax_split_vars to cmax_multiply_by_frac to more clearly describe what the stencil does.

I have not spent the time trying to add a docstring to cmax_split_vars, because I expect it to be changed when we merge this stencil with adjacent stencils, and this documentation would take much more time than this code change took.

## Code changes:

- Removed `mult_frac` from tracer_2d_1l.py
- Renamed cmax_split_vars to cmax_multiply_by_frac

## Checklist
Before submitting this PR, please make sure:

- [x] You have followed the coding standards guidelines established at [Code Review Checklist](https://paper.dropbox.com/doc/Code-Review-Checklist--BD7zigBMAhMZAPkeNENeuU2UAg-IlsYffZgTwyKEylty7NhY).
- [x] Docstrings and type hints are added to new and updated routines, as appropriate
- [x] All relevant documentation has been updated or added (e.g. README, CONTRIBUTING docs)
- [x] Unit tests are added or updated for non-stencil code changes

### CODE REVIEW

File: fv3core/stencils/tracer_2d_1l.py [hunk 1]
```diff
@@ -34,23 +34,17 @@ def flux_y(cy: sd, dya: sd, dx: sd, sin_sg4: sd, sin_sg2: sd, yfx: sd):
...
-    var_tmp = var
-    return var_tmp * frac
-
-
 @gtstencil()
-def cmax_split_vars(
+def cmax_multiply_by_frac(
     cxd: sd, xfx: sd, mfxd: sd, cyd: sd, yfx: sd, mfyd: sd, frac: float
 ):
     with computation(PARALLEL), interval(...):
```
Comment (jdahm) [inline fv3core/stencils/tracer_2d_1l.py:41]: Could use a docstring here.

Comment (mcgibbon) [inline fv3core/stencils/tracer_2d_1l.py:41]: I'll add this simple one for now. Documenting these inputs needs to be done along with some renames and generally an overhaul of this file.
```suggestion
def cmax_multiply_by_frac(
    cxd: sd, xfx: sd, mfxd: sd, cyd: sd, yfx: sd, mfyd: sd, frac: float
):
    """multiply all other inputs in-place by frac."""
    with computation(PARALLEL), interval(...):
```

Comment (mcgibbon) [reply fv3core/stencils/tracer_2d_1l.py]: Added a simple one for now. Documenting these inputs needs to be done along with some renames and generally an overhaul of this file.

**Review by jdahm — APPROVED** [review]
LGTM

**Pull request merged**